src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentPrice = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days <= 2) {
+    return rentPrice * days;
```

Comment:
You use `rentPrice * days` 3 times, so move it to variable

Reply:
I've changed this, thank you!